### PR TITLE
Govcy text inputs

### PR DESCRIPTION
**Pull request by astroman75**
This pull request concerns both the alerts and text inputs.

### CODE REVIEW

File: build/index.html [hunk 1]
```diff
@@ -7,12 +7,14 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
     <link href="css/main.css" rel="stylesheet">
-    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.min.js" integrity="sha384-cVKIPhGWiC2Al4u+LWgxfKTRIcfu0JTxR+EQDz/bgldoEyl4H0zUF0QKbrJ0EcQF" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.min.js"
```
Comment (benvand) [inline build/index.html:10]: I don't think we need this import at all actually 🤔 
Because we are importing the components directly

**Comment by benvand** [comment]
`build/css/main.css` is up to date 👍

File: build/index.html [hunk 2]
```diff
@@ -37,44 +39,63 @@ <h2>Headers:</h2>
             <li><a href="#">Login</a></li>
          </ul>
      </header>
+
```
Comment (benvand) [inline build/index.html:42]: Quite a lot of whitespace? Not needed?

File: src/sass/components/govcy-alerts.scss [hunk 3]
```diff
@@ -68,10 +68,10 @@
     @extend .alert;
```
Comment (benvand) [inline src/sass/components/govcy-alerts.scss:68]: I think there is a small problem with the `govcy-all-alerts` mixin. It looks like it's rendering a small white line at the edge. See screenshot

Comment (benvand) [reply src/sass/components/govcy-alerts.scss:68]: Screenshot 2021-12-13 at 10 10 25

File: src/sass/components/govcy-alerts.scss [hunk 4]
```diff
@@ -68,10 +68,10 @@
...
     @include govcy-all-alerts();
     @include govcy-alert-success-important();
-    border-color: olivedrab;
+    border-color: $govcy-success;
```
Comment (benvand) [inline src/sass/components/govcy-alerts.scss:71]: Loving the new variables 🙌

File: src/sass/components/govcy-text-inputs.scss [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+    input[type="time"]:focus,
+    input[type="week"]:focus,
+    input[type="number"]:focus,
+    input[type="email"]:focus,
+    input[type="url"]:focus,
+    input[type="search"]:focus,
+    input[type="tel"]:focus,
+    input[type="color"]:focus,
+    .uneditable-input:focus {   
+       border-color: gray;
```
Comment (benvand) [inline src/sass/components/govcy-text-inputs.scss:21]: I think this should be black maybe?

The figma prototypes have a black border
https://www.figma.com/file/zjXeMiVsDDt1QO8te0hoob/DMRID-%E2%80%93-Components?node-id=54%3A65

Comment (benvand) [reply src/sass/components/govcy-text-inputs.scss:21]: Screenshot 2021-12-13 at 10 14 05
Screenshot 2021-12-13 at 10 14 14

File: src/sass/components/govcy-text-inputs.scss [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+    input[type="tel"]:focus,
+    input[type="color"]:focus,
+    .uneditable-input:focus {   
+       border-color: gray; 
+      box-shadow: 0 5px 5px rgba(0, 0, 0, 0.075) inset, 0 0 8px rgba(126, 239, 104, 0.6);
+      outline: 5px solid $govcy-outline-color;
+      border-radius: $border-radius;
+    }
+
+  @mixin govcy-text-inputs-borders-fonts {
```
Comment (benvand) [inline src/sass/components/govcy-text-inputs.scss:27]: Nice use of the `@mixin` s

File: src/sass/components/govcy-text-inputs.scss [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+        margin-left:0px;
+        padding-left: 0px;
+        color:$govcy-danger;
+        font-weight: bold;
+        font-size: x-large;
+    }
+
+}
+
+.govcy-text-input-width-40em
```
Comment (benvand) [inline src/sass/components/govcy-text-inputs.scss:111]: Hahaha keeping it simple
If thats what they asked for why not

File: src/sass/components/govcy-text-inputs.scss [hunk 8]
```diff
@@ -0,0 +1,132 @@
...
+.govcy-text-input-width-10em
+{
+    width: 10em;
+    
+}
+
+.govcy-text-input-width-5em
+{
+    width: 5em;
+
```
Comment (benvand) [inline src/sass/components/govcy-text-inputs.scss:130]: Some extra spaces and newlines

File: src/sass/components/govcy-text-inputs.scss [hunk 9]
```diff
@@ -0,0 +1,132 @@
...
+    border-color: gray;
+    border-radius: $border-radius;
+    font-size: x-large;
+  }  
+
+.govcy-text-input-base {
+   
+    @extend .form-control;
+    @extend .form-text;
+    @include govcy-text-inputs-borders-fonts();
```
Comment (benvand) [inline src/sass/components/govcy-text-inputs.scss:39]: I don't _think_ you need the brackets `()`
https://sass-lang.com/documentation/at-rules/mixin
Could be wrong though. My sass is a bit out of date

**Comment by benvand** [comment]
There's something we can work out here about whether these need to be in a `form-group` wrapper/ Is form group good to use? Better for accessibility maybe or screen readers?

**Comment by UI-TEC** [comment]
Need code refactoring through new typography styles and the utility classes

**Pull request closed without merging**